zone/gm_commands/parcels.cpp

```diff
@@ -4,304 +4,488 @@
-		for (auto const &p: parcels) {
-			c->Message(
-				Chat::Yellow,
-				fmt::format(
-					"Slot [{:02}] has item id [{:10}] with quantity [{}].",
-					p.second.slot_id,
-					p.second.item_id,
-					p.second.quantity
-				).c_str()
-			);
-		}
-	}
-	if (is_add) {
-		if (arguments < 4) {
-			SendParcelsSubCommands(c);
-			return;
-		}
 
-		if (!Strings::IsNumber(sep->arg[3]) || !Strings::IsNumber(sep->arg[4])) {
-			SendParcelsSubCommands(c);
-			return;
-		}
+		std::string note;
+		uint16      note_index    = 5;
+		uint16      augment_index = 0;
 
-		auto to_name  = std::string(sep->arg[2]);
-		auto item_id  = Strings::ToUnsignedInt(sep->arg[3]);
-		auto quantity = Strings::ToUnsignedInt(sep->arg[4]);
-		auto note     = std::string(sep->argplus[5]);
+		if (arguments >= note_index) {
+			while (sep->IsNumber(note_index)) {
+				augment_ids[augment_index] = Strings::ToUnsignedInt(sep->arg[note_index]);
+
+				augment_index++;
+				note_index++;
+			}
+
+			note = sep->argplus[note_index];
+		}
 
 		auto send_to_client = CharacterParcelsRepository::GetParcelCountAndCharacterName(
 			database,
 			to_name
 		);
-		if (send_to_client.at(0).character_name.empty()) {
-			c->MessageString(Chat::Yellow, CANT_FIND_PLAYER, to_name.c_str());
+
+		const std::string& character_name = send_to_client.front().character_name;
+		const uint32 character_id = send_to_client.front().char_id;
+
+		if (character_name.empty()) {
+			c->MessageString(Chat::White, CANT_FIND_PLAYER, to_name.c_str());
 			return;
 		}
 
-		auto next_slot = c->FindNextFreeParcelSlot(send_to_client.at(0).char_id);
-		if (next_slot == INVALID_INDEX) {
+		const int free_slot = c->FindNextFreeParcelSlot(character_id);
+		if (free_slot == INVALID_INDEX) {
 			c->Message(
-				Chat::Yellow,
+				Chat::White,
 				fmt::format(
 					"Unfortunately, {} cannot accept any more parcels at this time. Please try again later.",
-					send_to_client.at(0).character_name
+					character_name
 				).c_str()
 			);
 			return;
 		}
 
+		const std::string& note_string = (
+			!note.empty() ?
+			fmt::format(
+				" with a note of '{}'",
+				note
+			) :
+			""
+		);
+
 		if (item_id == PARCEL_MONEY_ITEM_ID) {
-			if (quantity > INT32_MAX) {
+			if (quantity > std::numeric_limits<int>::max()) {
 				c->Message(
-					Chat::Yellow,
-					"Your quantity of {} copper pieces was too large.  Set to max quantity of {}.",
-					quantity,
-					INT32_MAX
+					Chat::White,
+					fmt::format(
+						"Your quantity of {} Copper was too large. Set to max quantity of {} Copper.",
+						Strings::Commify(quantity),
+						Strings::Commify(std::numeric_limits<int>::max())
+					).c_str()
 				);
-				quantity = INT32_MAX;
+				quantity = std::numeric_limits<int>::max();
 			}
 
 			auto item = database.GetItem(PARCEL_MONEY_ITEM_ID);
 			if (!item) {
-				c->Message(Chat::Yellow, "Could not find item with id {}", item_id);
+				c->Message(Chat::White, "Item ID {} does not exist.", item_id);
```

Comment:
Are there standards as to which color is for what?  Yellow vs white?  And the syntax of the messages?